selfdrive/controls/lib/planner.py

```diff
@@ -453,8 +453,8 @@ def update(self, CS, LoC, v_cruise_kph, user_distracted):
     events = []
     if self.model_dead:
       events.append(create_event('modelCommIssue', [ET.NO_ENTRY, ET.IMMEDIATE_DISABLE]))
-    if self.radar_dead or 'commIssue' in self.radar_errors:
-      events.append(create_event('radarCommIssue', [ET.NO_ENTRY, ET.IMMEDIATE_DISABLE]))
+    # if self.radar_dead or 'commIssue' in self.radar_errors:
+    #   events.append(create_event('radarCommIssue', [ET.NO_ENTRY, ET.IMMEDIATE_DISABLE]))
```

Comment:
@rickbias The radar interfaces are specific to each manufacturer's directory, but we have a different radar, we're also seeing this on the toyotas where some have continentals. Would it make sense to have a radar type in interface.py?